internal/pkg/workspace/workspace.go

```diff
@@ -157,10 +158,15 @@ func Create(appName string, fs afero.Fs) (*Workspace, error) {
 	return ws, nil
 }
 
+// ProjectRoot returns a path to the presumed root of the project, the directory that contains the copilot dir and .workspace file.
+func (ws *Workspace) ProjectRoot() (string) {
+	return strings.TrimSuffix(ws.CopilotDirAbs, "/copilot")
```

Comment:
We can use [`filepath.Dir`](https://pkg.go.dev/path/filepath#Dir) to handle forward/backward slash here!